buildings.mss

```diff
@@ -12,29 +15,41 @@
   [zoom >= 13] {
     polygon-fill: @building-low-zoom;
     polygon-clip: false;
+    line-width: 0;
     [zoom >= 15] {
-      line-color: @building-line;
       polygon-fill: @building-fill;
+      line-color: @building-line;
       line-width: .75;
       line-clip: false;
     }
-  }
-}
-
-#buildings-major {
-  [zoom >= 13] {
-    [aeroway = 'terminal'],
     [amenity = 'place_of_worship'],
-    [building = 'train_station'],
+    [aeroway = 'terminal'],
     [aerialway = 'station'],
+    [building = 'train_station'],
     [public_transport = 'station'] {
       polygon-fill: @building-major-fill;
-      polygon-clip: false;
-      [zoom >= 15] {
-        line-width: .75;
-        line-clip: false;
-        line-color: @building-major-line;
-      }
+      line-color: @building-major-line;
+    }
+    [building = 'roof'],
+    [building = 'construction'],
+    [building = 'damaged'],
+    [building = 'collapsed'],
+    [building = 'underground'],
+    [building = 'service'],
+    [building = 'slurry_tank'],
+    [building = 'shed'],
+    [building = 'garage'],
+    [building = 'garages'],
+    [building = 'farm_auxiliary'],
+    [building = 'carport'],
+    [building = 'barn'],
+    [building = 'stable'],
+    [building = 'cowshed'],
+    [building = 'greenhouse'],
+    [building = 'ger'],
```

Comment:
What is that?

I miss =hut and perhaps =cabin in the list.

Reply:
You mean ger? See https://en.wikipedia.org/wiki/Yurt